src/app/api/order/verify/route.ts

```diff
@@ -0,0 +1,37 @@
+import { NextRequest, NextResponse } from 'next/server'
+import crypto from 'crypto'
+
+const generatedSignature = (
+  razorpayOrderId: string,
+  razorpayPaymentId: string
+) => {
+  const keySecret = process.env.NEXT_PUBLIC_RAZORPAY_SECRET
```

Comment:
`process.env.NEXT_PUBLIC_RAZORPAY_SECRET` exposes `razor pay` secret to frontend. Should not be the case.